Plugins/VendorSQLite/AsyncNotOmittingEmptySubsequencesLineSequence.swift

```diff
@@ -0,0 +1,51 @@
+#if canImport(Darwin)
+
+import Foundation
+
+struct AsyncNotOmittingEmptySubsequencesLineSequence<Base: AsyncSequence>: AsyncSequence where Base.Element == UInt8 {
+    typealias Element = String
+    var base: Base
+    struct AsyncIterator: AsyncIteratorProtocol {
+        var src: Base.AsyncIterator, buf: Array<UInt8> = [], save: UInt8? = nil
+
+        @_specialize(where Base == FileHandle.AsyncBytes)
+        @_specialize(where Base == URL.AsyncBytes)
+        mutating func next() async rethrows -> String? {
+            let _CR:   UInt8 = 0x0D, _LF:   UInt8 = 0x0A
+            let _NEL1: UInt8 = 0xC2, _NEL2: UInt8 = 0x85
+            let _SEP1: UInt8 = 0xE2, _SEP2: UInt8 = 0x80, _SEP3L: UInt8 = 0xA8, _SEP3P: UInt8 = 0xA9
+
+            func yield() -> String? {
+                defer { buf.removeAll(keepingCapacity: true) }
+                return buf.isEmpty ? nil : String(decoding: buf, as: UTF8.self)
+            }
+            func nextByte() async throws -> UInt8? { defer { save = nil }; if let save { return save } else { return try await src.next() } }
```

Comment:
Perhaps a little bit too long for a one-liner?

Reply:
Exceeds even the 120 limit (141). I personally prefer 100 as i work on a 16 inch MacBook.

Reply:
Yeah, that's because it was originally shorter thanks to the `return if ...` syntax, but the 5.7 compiler is so brainless it wouldn't even parse the file no matter how many and which `#if`s I put around it. (And before that it was even shorter because I was trying to use `??`, which was never gonna work with an async call on the right-hand side.)

Reply:
But, there ya go, I addressed the complaint because it was reasonable, roll my eyes as I might 😉